pkg/rancher-desktop/backend/backendHelper.ts

```diff
@@ -217,4 +219,69 @@ export default class BackendHelper {
 
     return availableVersions[0];
   }
+
+  /**
+   * Return a dictionary of all containerd shims installed in /usr/local/bin.
+   * Keys are the shim names and values are the filenames.
+   */
+  static async containerdShims(): Promise<Record<string, string>> {
+    const shims : Record<string, string> = {};
+
+    try {
+      for (const filename of await fs.promises.readdir('/usr/local/bin')) {
```

Comment:
This is reading `/usr/local/bin` on the host?

Reply:
Yeah, that is not going to work then...

Reply:
I was wondering why this worked earlier today, but it seems like I was scanning the cache directory on the host at that time.